src/ape/api/networks.py

```diff
@@ -21,6 +22,10 @@
 LOCAL_NETWORK_NAME = "local"
 
 
+class ProxyInfoAPI(BaseModel):
```

Comment:
right now, `ProxyInfo` is not showing up in the docs. But if you a class doc to it, it does, something like

```python
    """
    Information about a proxy contract.
    """
```

And to get the `target` field to show up, you also have to add a doc, something like:

```
    """The address to the implementation contract."""
```